### DISCUSSION TITLE
Build error: found pages without a React Component

### DISCUSSION THREAD

**Discussion by Nfinished** [vercel/community#151] [Help] [answered]
Hi, my builds on Vercel are failing with the following error:
```
> Build error occurred
--
12:31:24.978 | Error: Build optimization failed: found pages without a React Component as default export in
12:31:24.978 | pages/projects/[id]
12:31:24.978 | pages/projects
12:31:24.978 |  
12:31:24.978 | See https://nextjs.org/docs/messages/page-without-valid-component for more info.
12:31:24.980 |  
12:31:24.980 | at /vercel/path0/node_modules/next/dist/build/index.js:604:19
12:31:24.981 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:60:20)
12:31:25.060 | error Command failed with exit code 1.


```

I'm a little stumped because
1. These pages absolutely do have default exported React components
2. My app runs and builds locally

What could be causing this? the only files in each folder are the `index.tsx` page component.

Node v14.17.6
Next v11.1.0

**Comment by Pieparker** [comment, marked as answer]
Hey there! Is this still an issue for you?

If so, please provide as much information as you can share – deployment IDs, commit SHAs, URLs and your account slug go a long way!

**Reply by Pieparker** [reply]
This conversation appears to have gone quiet. To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
